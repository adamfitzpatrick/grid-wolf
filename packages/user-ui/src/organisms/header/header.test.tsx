import React from 'react';
import { Header } from ".";
import { render, screen } from '@testing-library/react';

describe('Header', () => {
  let container: HTMLElement;

  beforeEach(() => {
  });

  test('should contain the account menu button', () => {
  });
});
