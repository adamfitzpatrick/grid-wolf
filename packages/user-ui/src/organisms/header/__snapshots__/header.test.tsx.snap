// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should contain the account menu button 1`] = `
<div>
  <div
    class="header"
  >
    <svg
      class="logo"
      version="1.1"
      viewBox="0 0 17.8 20.8"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        id="logo"
        transform="translate(0,0)"
      >
        <path
          d="M 0.3 5.3 l 8.66 -5 l 8.66 5 v 10 l -8.66 5 l -8.66 -5 v -10 z"
          fill="none"
          id="vert-hex"
          stroke="#5cdbd3"
          stroke-width="0.5"
        />
        <text
          fill="#722ed1"
          transform="scale(0.6)"
          x="2"
          y="22.5"
        >
          GW
        </text>
      </g>
    </svg>
    <div
      class="account-menu"
    >
      <button
        class="button primary large icon-only"
        title="account menu"
      >
        <span>
          <span
            aria-label="user"
            class="anticon anticon-user"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="user"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
              />
            </svg>
          </span>
        </span>
        <span />
      </button>
    </div>
  </div>
</div>
`;
